### DISCUSSION TITLE
Trivy CLI 0.54.1 scan only 1 specific deployment

### DISCUSSION THREAD

**Discussion by wicastk** [aquasecurity/trivy#7307] [Q&A]
### Question

Hi, I was reviewing Trivy documentation and the Trivy Kubernetes new experience changes, but I can't find the way to scan only 1 specific deployment using Trivy CLI 0.54.1 version, is it possible? any guidance? 
Thanks in advance :) 

### Target

Kubernetes

### Scanner

Vulnerability

### Output Format

None

### Mode

Standalone

### Version

```bash
Version: 0.54.1
```

**Comment by afdesk** [comment]
Hi @wicastk 
thanks for your interest in Trivy.

It seems there is a similar feature request - #6660.
You can track it there.

Now you select Deployments, but you already saw it in the docs:
```sh
$ trivy k8s --include-kinds Deployment --report summary
```

cc: @itaysk
